### PR TITLE
chore: Add Makefile with language targets

### PR DESCRIPTION
**Pull request by findkim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+```bash
+make golang
+```
+
+## Client Packages
+
+NVIDIA offers Riva client packages which include various language-specific bindings for the protocol buffers:
+
+* **C++**: [nvidia-riva/cpp-clients](https://github.com/nvidia-riva/cpp-clients)
+* **Python**: [nvidia-riva/riva-python-clients](https://github.com/nvidia-riva/riva-python-clients)
```
Comment (ryanleary) [inline README.md]:
```suggestion
* **Python**: [nvidia-riva/riva-python-clients](https://github.com/nvidia-riva/python-clients)
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,30 @@
...
+
+## Contributing
+
+Submit [GitHub issues](https://github.com/nvidia-riva/common/issues) for bugs and requests.
+
+Visit the [NVIDIA developer forum](https://forums.developer.nvidia.com/c/ai-data-science/deep-learning/riva/475) for questions and discussions.
```
Comment (ryanleary) [inline README.md]:
```suggestion
Visit the [NVIDIA Riva developer forum](https://forums.developer.nvidia.com/c/ai-data-science/deep-learning/riva/475) for questions and discussions.
```

**Review by ryanleary — COMMENTED** [review]
Generally LGTM! Couple readme fixes suggested.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,30 @@
...
 
-Riva clients use [gRPC](https://grpc.io) to communicate with the Riva server. The APIs and message types are defined in `riva/proto`. Complete API documentation is available in the [Riva user documentation](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/protobuf-api/protobuf-api-root.html).
\ No newline at end of file
+Riva clients use [gRPC](https://grpc.io) to communicate with the Riva server. The APIs and message types are defined in `riva/proto`. Complete API documentation is available in the [Riva user documentation](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/protobuf-api/protobuf-api-root.html).
+
+### Compile Protocol Buffers
+
+Compling `.proto` files require the protocol buffer compiler [`protoc`](https://developers.google.com/protocol-buffers/docs/downloads) and the corresponding [gRPC Codegen Plugin](https://grpc.io/docs/languages/) of your desired language.
+
+Use the [Makefile](Makefile) or refer to the official [Protocol Buffers](https://developers.google.com/protocol-buffers/docs/tutorials) to compose the `protoc` command.
```
Comment (findkim) [inline README.md]:
```suggestion
Use the [Makefile](Makefile) or refer to the [Protocol Buffers](https://developers.google.com/protocol-buffers/docs/tutorials) documentation to compose the `protoc` command.
```

**Pull request merged**